client/components/Authentication/RegisterContainer.jsx

```diff
@@ -107,12 +125,32 @@ const RegisterContainer = ({ navigation }) => {
   const submitForm = () => {
     if (isValidForm()) {
       // add user data to mongoDB
-      // add email and password to firebase authentication
-      handleSignUp(auth, email, password);
+      postUserData(
+        'http://localhost:3000/register',
```

Comment:
You might want to send this up to the AWS server as well. I noticed that expo doesn't work on my phone as well because it does not have a localhost booted up.

Reply:
has the mongodb connection been updated on the aws server?

Reply:
Yes, it's all hooked up to the remote mongodb. If you change the server, I have to git pull though. I *hope* to have the server continuously deployed by tomorrow night.